tethysext/atcore/public/resource_workflows/report_workflow_results_view.js

```diff
@@ -1,11 +1,73 @@
+                        fill: new ol.style.Fill({
+                                color: 'red',
+                              }),
+                        stroke: new ol.style.Stroke({color: 'red', width: 1}),
+                    }),
+        }),
+        'LineString': new ol.style.Style({
+            stroke: new ol.style.Stroke({
+                color: 'red',
+                width: 2,
+            }),
+        }),
+        'Polygon': new ol.style.Style({
+            stroke: new ol.style.Stroke({
+                color: 'red',
+                width: 2,
+            }),
+            fill: new ol.style.Fill({
+                color: 'rgba(255, 0, 0, 0.1)',
+            }),
+        }),
+    }
+    var style_function = function (feature) {
+        return styles[feature.getGeometry().getType()];
+    };
+
+    var base_layers = [
+        new ol.layer.Tile({
+            source: new ol.source.XYZ({
+                attributions: 'Tiles © <a href="https://services.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer">ArcGIS</a>',
+                url: 'https://server.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}',
+            }),
+        }),
+        new ol.layer.Tile({
+            source: new ol.source.XYZ({
+                attributions: 'Tiles © <a href="https://services.arcgisonline.com/ArcGIS/rest/services/Reference/World_Boundaries_and_Places/MapServer">ArcGIS</a>',
+                url: 'https://server.arcgisonline.com/ArcGIS/rest/services/Reference/World_Boundaries_and_Places/MapServer/tile/{z}/{y}/{x}',
+            }),
+        })]
     $('.resultType-map').each(function(i, obj) {
-        wmsSources[i] = new ol.source.ImageWMS({url: $(obj).data("layer-url"),
-            params: JSON.parse($(obj).data("layer-params").replace(/'/g, '"')),
-            ratio: 1,
-            serverType: 'geoserver',
-        });
+        layers = [...base_layers]
+        var layer_extent = ''
+        var map_data = $(obj).data('map-layer-variables')
+        map_data.forEach(function(data) {
+            if (data['source'] == 'TileWMS' || data['source'] == 'ImageWMS') {
+                wms_sources[i] = new ol.source.ImageWMS({url: data.options.url,
+                    params: JSON.parse(JSON.stringify(data.options.params).replace(/'/g, '"')),
+                    ratio: 1,
+                    serverType: 'geoserver',
+                });
+                layers.push(new ol.layer.Image({source: wms_sources[i]}))
+            }
+            if (data['source'] == 'GeoJSON') {
+                layers.push(new ol.layer.Vector({
+                                    style: style_function,
+                                    source: new ol.source.Vector({
+                                        features: new ol.format.GeoJSON({featureProjection: 'EPSG:3857'}).readFeatures(data.options),
+                                    })
+                                })
+                            )
```

Comment:
Unindent so that last parenthesis is in line with the beginning of `layers.push`.
Also missing semi-colon at the end: `);`